nipap/nipap-passwd

```diff
@@ -13,27 +13,29 @@ from nipap.nipapconfig import NipapConfig, NipapConfigError
 
 if __name__ == '__main__':
 
-    # parse options
-    parser = optparse.OptionParser()
-    parser.add_option('-a', '--add', dest='add_user', type='string', help='add user ADD_USER')
-    parser.add_option('-c', '--config', dest='config', default='/etc/nipap/nipap.conf', type='string',
-            help='read configuration from CONFIG [default:/etc/nipap/nipap.conf]')
-    parser.add_option('--create-database', action='store_true',
+    # parse arguments
+    parser = argparse.ArgumentParser(description='NIPAP User configuration')
+    parser.add_argument('action',
+        metavar='{list, add, delete, test-user, latest-version, create-database, upgrade-database}',
+        nargs='?', type=str,
+        choices=['list', 'add', 'delete', 'test-user', 'latest-version', 'create-database', 'upgrade-database'],
+        help='define an action to execute')
+    parser.add_argument('-u', '--user', dest='user', type=str, help='username')
+    parser.add_argument('-p', '--password', dest='password', type=str, help='set user\'s password to PASSWORD')
+    parser.add_argument('-n', '--name', dest='name', type=str, help='set user\'s name to NAME')
+    parser.add_argument('-t', '--trusted', action='store_true', dest='trusted', default=False, help='mark user as trusted')
+    parser.add_argument('-r', '--readonly', action='store_true', dest='readonly', default=False, help='set user to read only')
+    parser.add_argument('-f', '--file', dest='db_file', type=str, help="database file [default: read from config]")
+    parser.add_argument('-c', '--config', dest='config', default='/etc/nipap/nipap.conf', type=str, help=
+            'read configuration from CONFIG [default:/etc/nipap/nipap.conf]')
+    parser.add_argument('--version', action='version', version='nipap-passwd version %s' % nipap.__version__)
+    parser.add_argument('--latest-version', action='store_true',
+            help='check that the sqlite database is of latest version')
+    parser.add_argument('--create-database', action='store_true',
```

Comment:
Should this be here? Haven't we covered this action on line 18? Same for latest-version and upgrade-database